### ISSUE TITLE
Error when bundling iOS

### ISSUE THREAD

**Issue by Jerakin** [defold/editor2-issues#2593]
### Expected behaviour
Throw build error

### Actual behaviour
Throws Exception with a "Report Error Screen"

### Steps to reproduce
1. Add paths to iOS icons within the game.project that doesn't exists
2. Bundle for iOS

<hr/>

<table>
<tr><td>Build time</td><td>2019-04-30T13:23:48.989857</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>b8b891be5ba3e650202848d8b84ddc2ca2ed513a</td></tr>
<tr><td>Defold engine sha</td><td>11b1e7662dd68172fca551c52cba248eea16a364</td></tr>
<tr><td>Defold version</td><td>1.2.152</td></tr>
<tr><td>Error</td><td><a href='https://sentry.io/defold/editor2/?query=id%3A"6e6c0e323e654936a451e550cefb4e8d"'>6e6c0e323e654936a451e550cefb4e8d</a></td></tr>
<tr><td>GPU</td><td>AMD Radeon Pro 560 OpenGL Engine</td></tr>
<tr><td>GPU Driver</td><td>2.1 ATI-1.60.26</td></tr>
<tr><td>Java version</td><td>11.0.1+13</td></tr>
<tr><td>OS arch</td><td>x86_64</td></tr>
<tr><td>OS name</td><td>Mac OS X</td></tr>
<tr><td>OS version</td><td>10.13.3</td></tr>
<table>

**Comment by Jerakin** [comment]
Bob doesn't work either

```

Exception in thread "main" com.dynamo.bob.CompileExceptionError
	at com.dynamo.bob.Project.build(Project.java:352)
	at com.dynamo.bob.Bob.main(Bob.java:511)
Caused by: java.lang.NullPointerException
	at java.io.FileOutputStream.write(FileOutputStream.java:313)
	at org.apache.commons.io.FileUtils.writeByteArrayToFile(FileUtils.java:2095)
	at org.apache.commons.io.FileUtils.writeByteArrayToFile(FileUtils.java:2078)
	at com.dynamo.bob.bundle.BundleHelper.getFallbackIconImage(BundleHelper.java:223)
	at com.dynamo.bob.bundle.BundleHelper.copyIosIcons(BundleHelper.java:236)
	at com.dynamo.bob.bundle.IOSBundler.bundleApplication(IOSBundler.java:297)
	at com.dynamo.bob.Project.bundle(Project.java:492)
	at com.dynamo.bob.Project.doBuild(Project.java:853)
	at com.dynamo.bob.Project.build(Project.java:344)
	... 1 more

```

**Comment by Jerakin** [comment]
Tried all different combinations within the bundle screen, tried bob with 152 and 151

**Comment by Jerakin** [comment]
I could build the work project with its profile and identity. So it is definetly something with my setup, just need to figure out what 😛

**Comment by matgis-king** [comment]
The error suggests something is going wrong when copying an icon image to the bundle. Perhaps the work project specifies all the icon sizes manually or something?

**Comment by Jerakin** [comment]
Yay! The iOS section had paths to icons that didn't exists

**Comment by matgis-king** [comment]
Great! Thanks for helping us figure this out!

**Comment by AGulev** [comment]
Solved in 1.2.157 https://forum.defold.com/t/defold-1-2-157-has-been-released/55935
